docs/readme-zephyr.md

```diff
@@ -256,8 +256,6 @@ Use the ``CONFIG_ENABLE_MGMT_PERUSER=y`` Kconfig option to enable the following
 
 Images uploaded by the serial recovery can be decrypted on the fly by using ECIES primitives described in the [ECIES encryption](encrypted_images.md#ecies-encryption) section.
 
-Enable support for this feature by using ``CONFIG_BOOT_SERIAL_ENCRYPT_EC256=y``.
```

Comment:
Can you explain here How to use this feature with your changes?

Reply:
I might just remove this section since it doesn't really apply anymore, if encryption is enabled then encrypted images can be used, regardless of type of encryption.

Reply:
Yes, page on encryption: https://docs.mcuboot.com/encrypted_images.html